src/main/java/org/elasticsearch/index/query/MatchAllQueryBuilder.java

```diff
@@ -64,15 +56,19 @@ public MatchAllQueryBuilder boost(float boost) {
         return this;
     }
 
+    /**
+     * @return the boost for this query.
+     */
+    public float getBoost() {
```

Comment:
If we want to go with pure getter, which is fine with me, we should probably change our setters too then for consistency? I think we have `public MatchAllQueryBuilder boost(float boost)` here. Not talking about the fluent thing, just the name of the method, either get and set or no get nor set prefix .